web/styles/left_sidebar.css

```diff
@@ -159,7 +159,7 @@
         }
     }
 
-    .inactive_stream:not(.active-filter) {
+    .inactive_stream:not(.top-left-active-filter) {
```

Comment:
This seems wrong? Isn't `active-filter` here in the preexisting context of filtering channels?